mailipy/gen.py

```diff
@@ -108,11 +108,11 @@ def main():
         sys.exit(1)
 
     # Read template file
-    with open(args.template, "r") as mdfile:
+    with open(args.template, "r", encoding='utf-8') as mdfile:
```

Comment:
I am wondering why was this changed: do you get a different behavior when the `encoding` parameter is `None`? Do you get an error?

I checked the [documentation for `open`](https://docs.python.org/3/library/functions.html#open) and it seems that by default the encoding value is guessed by calling [`locale.getpreferredencoding`](https://docs.python.org/3/library/locale.html#locale.getpreferredencoding) which, at least on my system, returns 'UTF-8', so in theory leaving this to `None` should be better as we wouldn't be forcing to always read in utf-8.

Reply:
I can't read utf8 csv and .md files with pip install mailipy， after add encoding=utf8 the script work.
perhaps some thing is wrong with my env?

Reply:
What does this return on your system?

```
>>> import locale
>>> locale.getpreferredencoding(False)
'UTF-8'
```

Reply:
>>> import locale
>>> locale.getpreferredencoding(False)
'cp936'

Reply:
OK so that's most likely why you're getting that error.

Then I think it's fine to enforce that any valid .md template must be utf-8 encoded. I am OK with this change.